### PR TITLE
soap.d.ts: Improve ClientSSLSecurty constructor parameters

### PR DESCRIPTION
**Pull request by aleung**
- [x] Prefer to make your PR against the `types-2.0` branch.
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).
- [] Run `npm run lint -- package-name` if a `tslint.json` is present.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/vpulim/node-soap/blob/master/lib/security/ClientSSLSecurity.js
- [] Increase the version number in the header if appropriate.

### CODE REVIEW

**Comment by aleung** [comment]
@nicoleWjie Per your request, I change your name with mine from the 'Definitions by' list.

**Comment by nicoleWjie** [comment]
I reviewed the changes,passed ,Thanks

File: soap/index.d.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // Type definitions for soap
 // Project: https://www.npmjs.com/package/soap
-// Definitions by: Nicole Wang <https://github.com/nicoleWjie>, Cage Fox <https://github.com/cagefox>
+// Definitions by: Leo Liang <https://github.com/aleung>, Cage Fox <https://github.com/cagefox>
```
Comment (vvakame) [inline soap/index.d.ts]: Why replace @nicoleWjie name?

Comment (aleung) [inline soap/index.d.ts]: See my previous comment:

> @nicoleWjie Per your request, I change your name with mine from the 'Definitions by' list.

She is my team mate. We use node-soap in our project. She doesn't want to maintain this definition any more and let me do it.

**Pull request merged**